modules/coremark/CMakeLists.txt

```
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(NOT CONFIG_COREMARK)
+  return()
+endif()
+
+zephyr_library_named(coremark)
+
+zephyr_include_directories(${ZEPHYR_COREMARK_MODULE_DIR} .)
+message((WARNING, " cormark modul dir:   ${ZEPHYR_COREMARK_MODULE_DIR}"))
```

```suggestion
message(WARNING, "coremark module dir:   ${ZEPHYR_COREMARK_MODULE_DIR}")
```

is it necessary or only for debug?